.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "files.autoSave": "off"
```

Comment:
Question: why would have you needed this ?

Reply:
This is due to this issue: https://github.com/statelyai/editor-feedback/issues/34.
Stately Editor does not play well with autosave, which I have always turned on usually.

Reply:
Ok thx !